compiler/luci-interpreter/pal/linux/PALGRU.h

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef LUCI_INTERPRETER_PAL_GRU_H
+#define LUCI_INTERPRETER_PAL_GRU_H
+
+#include <tensorflow/lite/kernels/internal/reference/fully_connected.h>
+#include "PALreference_ops.h"
+namespace luci_interpreter_pal
+{
+
+// tflite's Logistic does not provide inplace Logistic kernel
+void Logistic(const int flat_size, const float *input_data, float *output_data)
+{
+  const float cutoff_upper = 16.619047164916992188f;
```

Comment:
Is it enough to hold that constant as float?

Reply:
Maybe we lose the little accuracy depending on size of float. Here since we borrow the code from tflite, we will keep this as it is.